src/WorldWindow.js

```diff
@@ -979,6 +979,20 @@ define([
         };
 
         /**
+         * Indicates whether this world window contains a specified layer.
+         * @param {Layer} layer The layer to test.
+         * @returns {Boolean} true if this world window contains the specified layer, otherwise false.
+         */
+        WorldWindow.prototype.containsLayer = function (layer) {
```

Comment:
Could we use indexOf instead? It might be useful to get the index of a particular layer in the array as well. There is something like that in LayerList in WWA at https://github.com/NASAWorldWind/WorldWindAndroid/blob/develop/worldwind/src/main/java/gov/nasa/worldwind/layer/LayerList.java#L66